.github/workflows/ant.yml

```diff
@@ -0,0 +1,45 @@
+name: Make self-contained Windows EXE
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+    - v*
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
In case it runs on Linux we could move the various steps to a Shell file so that we don't end up with proprietary dependencies and gets local reproducibility.